docs/schemas/Controlled-Vocabularies/BICAN-Project-and-Data-Collections.csv

```diff
@@ -180,7 +180,7 @@ U19 Huang,zhuang_merfish_NAc_proj,zhuang_merfish_NAc,A molecularly defined and s
 U19 Zeng,zeng_tolias_pseq_proj,zeng_tolias_pseq_e,Patch-seq of mouse motor cortex: electrophysiology,Patch-clamp electrophysiology recordings.,http://identifiers.org/DANDI:000008,CCBY4
 U19 Zeng,zeng_tolias_pseq_proj,zeng_tolias_pseq_t,Patch-seq of mouse motor cortex: transcriptomics,Transcriptomics from patched cells.,http://data.nemoarchive.org/biccn/grant/zeng/tolias/,CCBY4
 U19 Zeng,zeng_tolias_pseq_proj,zeng_tolias_pseq_m,Patch-seq of mouse motor cortex: morphology,Morphological reconstructions.,http://download.brainimagelibrary.org/biccn/zeng/tolias/pseq/morph/,CCBY4
-U19 Zeng,zeng_sn_SSv4_proj,zeng_sc_SSv4,Transcriptomic cell types in the mouse brain: SMART-seq cells,Single nucleus characterization in mouse brain using SMART-seq v4.,http://data.nemoarchive.org/biccn/lab/zeng/transcriptome/sncell/SMARTer/,CCBY4
+U19 Zeng,zeng_sn_SSv4_proj,zeng_sc_SSv4,Transcriptomic cell types in the mouse brain: SMART-seq cells,Transcriptomic cell types in the mouse brain: SMART-seq cells,http://data.nemoarchive.org/biccn/lab/zeng/transcriptome/sncell/SMARTer/,CCBY4

```

Comment:
I think you copied the Collection Title, not the Collection Description here.

Reply:
I'm seeing first description as "Single cell characterization in mouse brain using SMART-seq v4." on line 23